### ISSUE TITLE
Upgrade `envoyproxy/go-control-plane` when deadlock is resolved

### ISSUE THREAD

**Issue by lobkovilya** [kumahq/kuma#9580]
### Description

This is an issue in Kuma to track https://github.com/envoyproxy/go-control-plane/issues/875

**Comment by lukidzi** [comment]
We are using fork for now kumahq/go-control-plane

**Comment by slonka** [comment]
@lukidzi so shouldn't we close this then? as the linked issue is closed as "not planned"

**Comment by lukidzi** [comment]
Yes, let's close it for now and reopen once we can get changes into go-control-plane

**Comment by lukidzi** [comment]
Let's reopen once we get things in go-control-plane
